samples/nextjs/.env

```diff
@@ -29,4 +29,9 @@ SITECORE_API_KEY=
 # not exist when building locally (if you've never run `jss setup`), or when building
 # in a higher environment (since `scjssconfig.json` is ignored from source control).
 # In this case, use this environment variable to provide the value at build time.
-SITECORE_API_HOST=
\ No newline at end of file
+SITECORE_API_HOST=
+
+# Your GraphQL Edge endpoint. This is required for Sitecore Experience Edge.
+# For Sitecore XM, this is typically optional. By default, the endpoint is calculated using
+# the resolved Sitecore API hostname + the `graphQLEndpointPath` defined in your `package.json`.
+GRAPH_QL_ENDPOINT=
```

Comment:
@anastasiya29 No, we haven't listed these anywhere other than on the the production deployment doc, since they aren't technically necessary until that time. But I do wonder if we should call them out as an alternative to using the `scjssconfig.json` (since you totally can with Next.js)? I can add a backlog item if you think we should.